app/templates/gentelella/users/events/event_base.html

```diff
@@ -107,16 +107,34 @@
                     <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
                             aria-hidden="true">&times;</span></button>
                     <h4 class="modal-title">{{ _("Are you sure you would like to delete this event") }} ?</h4>
+                    <span>{{ _("Deleting the event will delete all the data associated with it.") }} <br><br>{{ _("Please enter the full name of the event to continue") }}</span><br>
+                    <input id = 'delete-check' type = "text"/>
                 </div>
                 <div class="modal-footer">
                     <a type="button" class="btn btn-default" data-dismiss="modal">{{ _("Cancel") }}</a>
                     <a href="/events/{{ event.id }}/trash/" type="button"
-                       class="btn btn-danger">{{ _("Confirm Delete") }}</a>
+                       class="btn btn-danger" id = "delete-button">{{ _("Confirm Delete") }}</a>
                 </div>
             </div>
         </div>
     </div>
 {% endblock %}
 {% block tail_js %}
     {{ super() }}
+    <script type = "text/javascript">
+        $('#delete-button').addClass('disabled');
+        $('#delete-check').keyup(function(){
+            var x = $('#delete-check').val();
+            if(x == '{{ event.name }}')
+            {
+                $('#delete-button').removeClass('disabled');
+            }
+            else
+            {
+                if(!$('#delete-button').hasClass('disabled'))
```

Comment:
You're using `$('#delete-button')` multiple times. Which is inefficient. Store it in a variable and refer from that.